backend/src/infra/services/shared/token/jwt-token-manager.ts

```diff
@@ -0,0 +1,40 @@
+import { CustomError } from "@/application/error";
+import { StudentRepository } from "@/application/repositories/student-repository";
+import { StudentTokenManager } from "@/application/services/token/token-manager";
+import { Student } from "@/domain";
+import jwt from 'jsonwebtoken';
+
+export class JWTStudentTokenManager implements StudentTokenManager {
+
+    SECRET_KEY: string = "unbordoJWTsecretkey*4#$@);.";
```

Comment:
Nessa `SECRET_KEY` dá para colocar um valor no ENV assim poderemos mudar a secret key facilmente dependendo da plataforma que estamos rodando ficando até mesmo mais anônimo, o código do GitHub é publico, com essa chave damos a liberdade de qualquer usuário criar um token em nossa plataforma ou descriptografar um token de usuário da nossa plataforma, caracterizando uma falha de segurança.

```suggestion
    SECRET_KEY: string = env.JWT_SECRET_KEY
```

Após aplicar a sugestão será necessário adicionar no arquivo local de variável ambiente `.env.local` ou `.env.dev` e `.env.example`, além disso, alterar o arquivo `env.ts` localizado em `src/env.ts`